docs/cqd_reference/src/asciidoc/_chapters/query_execution.adoc

```diff
@@ -237,3 +237,27 @@ If you enable small scanner on large size scan incorrectly, then you are likely
 | *Deprecated In Release*     | Not applicable.
 |===
 
+<<<
+[[scratch-disks]]
+== SCRATCH_DISKS
+
+[cols="25%h,75%"]
+|===
+| *Description*               |	Allows a CQD specification to override the env settings for STFS locations.
```

Comment:
The text mentions "env settings". Can someone describe what that means? Is there a run-time environment variable that is consulted, for example? And if so, what's the name of that variable?